### DISCUSSION TITLE
fabric sample issue (container violation in fabric2-all-in-one)

### DISCUSSION THREAD

**Discussion by zondervancalvez** [hyperledger/fabric-samples#780] [General]
We are currently doing security scan for all our containers and vulnerability issues (CVE-2021-3121, CVE-2019-16884, CVE-2019-19921, CVE-2020-14040) that came up is https://github.com/hyperledger/cactus/issues/2057. As per investigation when we tried to upgrade the fabric versions and its dependencies based on [this](https://hyperledger-fabric.readthedocs.io/en/release-2.2/install.html) but still no changes in the scan. Is there a way that we can upgrade the versions of specific vulnerabilities mentioned above?

**Comment by davidkel** [comment]
This issue may be worth a read https://github.com/hyperledger/fabric/issues/3465
